lib/travis/build/script/haxe.rb

```diff
@@ -124,6 +125,20 @@ def script
 
         private
 
+          def haxe_versions
+            require 'faraday'
+            JSON.parse(Faraday.get("https://haxe.org/website-content/downloads/versions.json").body.to_s)
```

Comment:
Perhaps we should fall back to something reasonable if the HTTP request fails.

Reply:
Yes, it makes sense. I've just added that!